### DISCUSSION TITLE
Self-Service Portal – Sand-ID

### DISCUSSION THREAD

**Discussion by adn-misa** [e-id-admin/public-sandbox-trustinfrastructure#14] [Use Cases]
[https://ssi-start.adnovum.com](https://ssi-start.adnovum.com)

To help quickstart the [DIDAS community](https://didas.swiss), Adnovum created a Self-Service portal where everyone can issue themselves the following Base Identities: 

- **Sand-ID:** Representation of a governmental ID on the Sandbox for governmental processes. 
- **Employee ID:** Representation of a digital employee badge for the work environment
- **Social ID:** Self-proclaimed pseudo identity by the user for private use cases

&nbsp;

<img width="451" alt="image" src="https://github.com/e-id-admin/public-sandbox-trustinfrastructure/assets/68414331/d2fbcacd-9884-4e4a-b486-883b0bfebc75">

&nbsp;

Our goal was to make identity issuance in the sandbox more accessible and to help people start building use cases. Verifying these issued verifiable credentials is straightforward and can be done without additional prerequisites.

**Comment by CHempel-esatus** [comment]
Great, thank you so much for providing this. We have just tested this with our esatus Wallet and it works perfectly. If anyone else wants to test it, you can download the esatus Wallet here:

App Store:
[https://apps.apple.com/de/app/esatus-wallet/id1496769057](https://apps.apple.com/de/app/esatus-wallet/id1496769057)

Google Play Store:
[https://play.google.com/store/apps/details?id=com.esatus.wallet](https://play.google.com/store/apps/details?id=com.esatus.wallet)

![esatus_Wallet](https://github.com/e-id-admin/public-sandbox-trustinfrastructure/assets/53933121/fd9b5fe6-66bb-4657-88dd-af40ae43dd61)

**Reply by damienbod** [reply]
Thanks for this. Do you have a scheme definition for the Sand-ID? Is this the future E-ID ? 

Is the project open source? github link?

Kind regards Damien

**Reply by adn-misa** [reply]
Hi Damien,

Coming back to your questions:

Here are the scheme definition for our Sand-ID, Employee-ID and Social-ID:

- Sand-ID: 2xS9NY4w46MUCXEtAWT7Gf:2:SsiSspSandID:0.72
- Employee-ID: 2xS9NY4w46MUCXEtAWT7Gf:2:SsiSspEmployeeID:1.1
- Social-ID: 2xS9NY4w46MUCXEtAWT7Gf:2:SsiSspSocialID:1.0

Those IDs can be found on the Indy Scan service of the Public Sandbox: https://explorer.sandbox.ssi.ch/home/SANDBOX

This is not the future E-ID, this is a project to help people try the public sandbox and make it easier to integrate and test with their infrastructure.

The project is based on a frontend and our open-source Verifiable Credential Management System (VCMS). The frontend is quite simple and closed source but VCMS is available on GitHub(https://github.com/SSI-Solutions/vcms).

Regards,
Michel

**Reply by damienbod** [reply]
Hi @adn-misa  Thanks for your reply. 

I created an issue for my question here: 

https://github.com/e-id-admin/governance-sounding-board/discussions/49

**Reply by lanterno** [reply]
Hello @CHempel-esatus I'm facing a problem while trying to use the Esatus Android app with the adnovum issuer. 

Once I scan the QR code given by adnovum on the wallet, the connection is established correctly, however once the wallet received the VC offer. it freezes and starts displaying "not responding" messages. If I ignore the messages, the app continues to behave normally and I manage to accept the VC, but it still doesn't appear in the wallet.
Also, on adnovum side, the webpage never moves on from the "accept offer" loading state, so it seems it's excepting some kind of feedback which it's not getting.

I would appreciate it if you can look into this. Thanks!
